databricks-catalog-external-location/variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "tags" {
+  description = "Tags to apply to resources"
+  type        = map(string)
+}
+
+variable "bucket_name" {
+  description = "Name of the bucket"
+  type        = string
+}
+
+variable "databricks_external_location_iam_role" {
+  description = "Name of the role for instance profiles"
+  type        = string
+}
+
+variable "name_prefix" {
```

Comment:
this seems to only be used for the external location name - is the expectation that we just set this to the catalog name?